Neos.Flow/Classes/ResourceManagement/Target/SymlinkedFolderResourceObject.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\Flow\ResourceManagement\Target;
+
+use Neos\Flow\ResourceManagement\ResourceMetaDataInterface;
+
+/**
+ * @internal only for the callback {@see TargetInterface::onPublish()}
```

Comment:
Not entirely true, it is used above in Neos.Flow/Classes/ResourceManagement/Target/FileSystemSymlinkTarget.php – and that makes me wonder